client/src/modules/stock/requisition/templates/action.cell.html

```diff
@@ -6,7 +6,7 @@
 
   <ul data-action="{{ rowRenderIndex}}"  class="dropdown-menu" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
     <li>
```

Comment:
This should have the class `dropdown-header` on it to close IMA-WorldHealth#4424